core/txpool/blobpool/blobpool.go

```diff
@@ -1488,22 +1500,45 @@ func (p *BlobPool) Pending(filter txpool.PendingFilter) map[common.Address][]*tx
 					break // blobfee too low, cannot be included, discard rest of txs from the account
 				}
 			}
-			// Transaction was accepted according to the filter, append to the pending list
-			lazies = append(lazies, &txpool.LazyTransaction{
-				Pool:      p,
-				Hash:      tx.hash,
-				Time:      execStart, // TODO(karalabe): Maybe save these and use that?
-				GasFeeCap: tx.execFeeCap,
-				GasTipCap: tx.execTipCap,
-				Gas:       tx.execGas,
-				BlobGas:   tx.blobGas,
-			})
+			// Transaction was accepted according to the filter, append to the pending set
+			lazyTx := &txpool.LazyTransaction{
+				Pool:    p,
+				Hash:    tx.hash,
+				Time:    execStart,
+				Fees:    *tip,
+				Gas:     tx.execGas,
+				BlobGas: tx.blobGas,
+			}
+			if first {
+				first = false
+				tail = make([]*txpool.LazyTransaction, 0, len(txs)-i)
+				heads = append(heads, &txpool.TxFees{
+					From: addr,
+					Fees: lazyTx.Fees,
+				})
+			}
+			tail = append(tail, lazyTx)
+		}
+		if len(tail) > 0 {
+			tails[addr] = tail
 		}
-		if len(lazies) > 0 {
-			pending[addr] = lazies
+	}
+	return txpool.NewPendingSet(heads, tails)
+}
+
+// PendingHashes retrieves the hashes of all currently processable transactions.
+// The returned list is grouped by origin account and sorted by nonce
+func (p *BlobPool) PendingHashes(filter txpool.PendingFilter) []common.Hash {
```

Comment:
todo make use of the filter

Reply:
Is it ok if this method ignores the minTip? I think so, but it's unclean from api-perspective to take a filter but not respect it